test/games/strategy/triplea/xml/LoadGameUtil.java

```diff
@@ -49,10 +45,10 @@ private static GameData loadGame(final String game, final String[] possibleFolde
    * First try to load the game as a file on the classpath, if not found there
    * then try to load it from either the "maps" or "test_data" folders.
    */
-  private static InputStream openInputStream(final String game, final String[] possibleFolders) {
+  private static InputStream openInputStream(final String game, String[] possibleFolders) {
     InputStream is = LoadGameUtil.class.getResourceAsStream(game);
     if (is == null) {
-      final File f = GameRunner2.getFile(game, possibleFolders);
```

Comment:
I don't recall why it was removed, guessing it might have been a leaked in from other changes maybe. The `final` has been restored.